cube/src/main/java/zone/cogni/asquare/cube/index/IndexMethod.java

```diff
@@ -67,6 +67,29 @@ public Params getParams() {
   private final String indexName;
   private final SparqlSelectToJson sparqlSelectToJson;
 
+  public IndexMethod(ModelToJsonConversion modelToJsonConversion,
+                     String indexName,
+                     Elasticsearch7Store elasticStore) {
+    this(null, null, modelToJsonConversion, indexName, elasticStore, null);
+  }
+
+  public IndexMethod(ModelToJsonConversion modelToJsonConversion,
+                     String indexName,
+                     Elasticsearch7Store elasticStore,
+                     SparqlSelectToJson sparqlSelectToJson) {
+    this(null, null, modelToJsonConversion, indexName, elasticStore, sparqlSelectToJson);
+  }
+
+  @Deprecated
+  public IndexMethod(PaginatedQuery paginatedQuery,
```

Comment:
@marianna-tselepi since we are deprecating use of RdfStoreService here, can we also deprecate all methods which have graphUri as a parameter. Those are related.